fontir/src/ir.rs

```diff
@@ -135,6 +190,34 @@ impl Glyph {
     }
 }
 
+impl TryInto<Glyph> for GlyphBuilder {
+    type Error = WorkError;
+
+    fn try_into(self) -> Result<Glyph, Self::Error> {
+        Glyph::new(self.name, self.codepoints, self.sources)
+    }
+}
+
+impl From<&Glyph> for GlyphBuilder {
+    fn from(value: &Glyph) -> Self {
+        Self {
+            name: value.name.clone(),
+            codepoints: value.codepoints.clone(),
+            sources: value.sources.clone(),
+        }
+    }
+}
```

Comment:
I think we should delete this, and make the caller clone on their side, otherwise this can hide the fact that a clone is happening.

(and if we keep it, the impl can just be `value.clone().into()`)

Reply:
> this can hide the fact that a clone is happening.

Interesting. This doesn't reflect what I feel I see in Rust thus far, though ofc I would defer to your greater experience. To me it's no different than `From<&str> for String`: you definitely aren't getting an owned thing from a ref without some allocation, so it's clear from the signature there is some copying going on.

Reply:
Merging this but happy to send a follow-on PR to adjust if it becomes clear what I did was a Bad Idea.